src/bindings/js/node/src/helper.cpp

```diff
@@ -150,3 +152,21 @@ Napi::String cpp_to_js<ov::element::Type_t, Napi::String>(const Napi::CallbackIn
         }
     return str;
 }
+
+template <>
+ov::Tensor get_request_tensor(ov::InferRequest infer_request, std::string key) {
+    return infer_request.get_tensor(key);
+}
+
+template <>
+ov::Tensor get_request_tensor(ov::InferRequest infer_request, size_t idx) {
+    return infer_request.get_input_tensor(idx);
+}
+
+
+
+ov::Tensor value_to_tensor(Napi::Object obj){
+    // Check of object type
+    auto* tensor_wrap = Napi::ObjectWrap<TensorWrap>::Unwrap(obj); // here add check if this is a TensorWrap{}. Cannot be added in header because Napi::ObjectWrap<TensorWrap> requires the full definition of TensorWrap object
```

Comment:
```suggestion
    auto tensor_wrap = Napi::ObjectWrap<TensorWrap>::Unwrap(obj); // here add check if this is a TensorWrap{}. Cannot be added in header because Napi::ObjectWrap<TensorWrap> requires the full definition of TensorWrap object
```
I think auto should deduce it is pointer.

Is this comment required?
Is usually better to have functions not in headers.